unsplash/unsplash.go

```diff
@@ -59,6 +61,12 @@ func New(client *http.Client) *Unsplash {
 	return unsplash
 }
 
+func NewWithClientID(client *http.Client, client_id string) *Unsplash {
```

Comment:
Please add a Godoc string to this function. Please see other functions in the file.